datalad_registry/config.py

```diff
@@ -28,3 +28,12 @@ class DevelopmentConfig(Config):
 
 
 # TODO: ProductionConfig
+# from sqlalchemy.engine.url import URL
+# SQLALCHEMY_DATABASE_URI=str(URL.create(
+#     drivername = "postgresql",
+#     host       = "db",
+#     port       = 5432,
+#     database   = "dlreg",
+#     username   = "dlreg",
```

Comment:
how do you envision to setup the "production" (or local user testing) environment?  could we feed password somehow e.g. via env var or even interactively with some command?

Reply:
@yarikoptic 

> how do you envision to setup the "production" (or local user testing) environment?

I'm not clear on what you're asking

> could we feed password somehow e.g. via env var

Yes.

> or even interactively with some command?

That sounds like a terrible idea for production.

Reply:
> > how do you envision to setup the "production" (or local user testing) environment?
> 
> I'm not clear on what you're asking

an environment which is started by running `./up` should not use a hardcoded password

Reply:
@yarikoptic The environment started by `./up` now sets the PostgreSQL password via a `POSTGRES_PASSWORD` environment variable.